config/biggerreactors-server.toml

```diff
@@ -57,7 +57,7 @@ EnableAdvancedConfig = false
     # Valid range: (0,)
     # Default: 1.0
     #
-    OutputMultiplier = 1.0
+    OutputMultiplier = 3.0
```

Comment:
Does this affect just Reactors or also turbines? If it's just reactors, does this affect power output or does it also increase the rate at which water boils inside the reactor?

Reply:
Reactors from the mod Bigger Reactors will now produce 3x the power from before. This should not change how much power the Turbines create. -the post

Reply:
This might change the total amount heat produced, but doesn't effect the multiplier for output of the Turbine itself. Granted, more heat means you can power bigger turbines, but that doesn't mean the Turbine makes 3x more itself. From my testing, this puts the reactors in line with other power methods.

Reply:
Reactors only (turbines actually don't have a config that functions identically, i should add that)
Effects both active and passive reactors equal.

Notable: may want to also change the PassiveBatteryPerExternalBlock and CoolantTankAmountPerFuelRod options (hidden by default) to ensure those don't become limiting factors

Reply:
Forgot to also mention, this is added right at the end for both modes, so the output numbers change without anything else changing.

When double checking that I also found a bug with this in active reactors, passive creators will work as expected.

Reply:
I'm also also going to note that this change *could* be problematic for existing reactor/turbine setups if they don't auto-throttle themselves, idk if there is any good way to alert users to this change.

Reply:
I think just putting "If you have a Turbine/Reactor combo, check on it." Otherwise, thank you for checking this out!

Reply:
Also, what bug happened because I actually was struggling with a bug while testing this. My reactor's heat would just climb infinitely, and the turbine would not be able to spin up without issues.

Reply:
reactor could boil more coolant than it has in it, running the input tank negative.

a reactor's heat will climb a lot of if the coolant flow is too low, unlike passive reactors active ones wont self-stabilize at any semi-reasonable temperature

Reply:
That sounds like the issue I was having.